### ISSUE TITLE
NTR Pelger-Huet anomaly

### ISSUE THREAD

**Issue by diatomsRcool** [mgijax/mammalian-phenotype-ontology#3637]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)
Pelger-Huet anomaly

>2. Synonyms (e.g., absent spleen)

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

nuclei of several types of white blood cells (neutrophils and eosinophils) have unusual shape (bilobed, peanut or dumbbell-shaped) and unusual structure (coarse and lumpy).

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

Seen in dogs and humans PMID: 2649629

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

abnormal neutrophil morphology and abnormal eosinophil morphology

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

[0000-0002-5034-2637](https://orcid.org/0000-0002-5034-2637)

**Comment by sbello** [comment]
Additional reference https://pubmed.ncbi.nlm.nih.gov/22338047/ 
"The Pelger-Huët anomaly (PHA) is a recognized morphologic variant affecting all granulocytes but is most evident in polymorphonuclear neutrophils (PMNs)."

" It is now known that heterozygous mutations of LBR result in hyposegmented PMN nuclei known as PHA."
Table 1 has list of features of PHA and pseudo-PHA

So, it seems like the salient features are:

- hyposegmented, symmetrical nuclei
- more condensed chromatin

Just having hyposegmentation would not differentiate between PHA and pseudo-PHA, we could add new terms for both symmetrical hyposegmentation and increased chromatin condensation.

PMID:24255920 describes mouse model with pseudo-PHA in neutrophils

a number of existing annotations refer to hypersegmented nuclei in neutrophils

MGI has one model of PHA - mentions both neutrophil and eosinophil segmentation (PMID:12490533)

**Comment by sbello** [comment]
Proposal add terms for:
abnormal granulocyte nuclear morphology
  abnormal neutrophil nuclear morphology
     abnormal neutrophil nuclear segmentation
       decreased neutrophil nuclear segmentation
          symmetrical hyposegmentation of neutrophil nuclei (related synonym Pelger-Huet anomaly)
          asymmetrical hyposegmentation of neutrophil nuclei
       increased neutrophil nuclear segmentation
    abnormal neutrophil chromatin condensation
       increased neutrophil chromatin condensation
       decreased neutrophil chromatin condensation
  abnormal eosinophil nuclear morphology
      abnormal eosinophil nuclear segmentation
       decreased eosinophil nuclear segmentation (related synonym Pelger-Huet anomaly)
       increased eosinophil nuclear segmentation

**Comment by sbello** [comment]
@diatomsRcool How critical is it for you to have the combined term? My preference would be to follow the proposal above with separate terms for the neutrophils and eosinophils that have a synonym for PHA. This would avoid some of the confusion of the variable usage and presentation of PHA but would mean you would need to annotate to more terms.

**Comment by diatomsRcool** [comment]
I will defer to @hughesk83 who is the specialist here

**Comment by hughesk83** [comment]
I think this looks good but I am not sure if we should have chromatin condensation also included. I think that is more subjective and the better characterized phenotype is the hyposegmentation. I think having separate terms for neuts and eos is a good suggestion.

**Comment by sbello** [comment]
I'm fine not dealing with the chromatin terms. I'll try to get the rest in in the next week.

**Comment by sbello** [comment]
Added terms:

MP:0021171 | abnormal granulocyte nuclear morphology
MP:0021172 | abnormal neutrophil nuclear morphology
MP:0021173 | abnormal neutrophil nuclear segmentation
MP:0021174 | decreased neutrophil nuclear segmentation
MP:0021175 | symmetrical hyposegmentation of neutrophil nuclei
MP:0021176 | asymmetrical hyposegmentation of neutrophil nuclei
MP:0021177 | increased neutrophil nuclear segmentation
MP:0021178 | abnormal eosinophil nuclear morphology
MP:0021179 | abnormal eosinophil nuclear segmentation
MP:0021180 | decreased eosinophil nuclear segmentation
MP:0021181 | increased eosinophil nuclear segmentation

**Comment by sbello** [comment]
@diatomsRcool I've added all the terms but the MP build is currently failing due (I think) to unrelated issues. We're looking into the issue and I'll let you know when the terms are available for use.

**Comment by sbello** [comment]
Build is back up, terms should be available on Monday
